src/Connector.SqlServer/Utils/TableDefinitions/MainTableDefinition.cs

```diff
@@ -94,6 +94,11 @@ public static MainTableColumnDefinition[] GetColumnDefinitions(StreamMode stream
                                 return entityTypeValue.ToString();
                             }
 
+                            if (propertyValue is PersonReference personReference)
+                            {
+                                return personReference.Name;
+                            }
```

Comment:
PersonReferences is an EntityReference. It is not necessarily just a name. It could be an entity code as well.
Ie. there might not be a name, but only a code.

Reply:
If we are dealing with PersonReferences then we should properly also add handling for tags and EntityReference as well.

Reply:
Changed to handle EntityReference, prefering codes over names if available.
Tags already works:
![{C9A80FA7-29BC-4C3E-98DB-A042F684AB17}](https://github.com/user-attachments/assets/e8dabb40-9541-40ca-a53e-6343e6e8e313)